src/include/partitioning/custom_part_check_generated.rb

```diff
@@ -294,6 +299,20 @@ def check_created_partition_table(targetMap, installation)
         ok = false if !Popup.YesNo(message)
       end
 
+      # A PReP/CHRP partition is not supposed to be mounted. So if we find any
+      # other /boot partition, we should warn the user.
+      if boot_found && Partitions.PrepBoot && !boot_mount_point.empty? && !Arch.board_iseries && installation || show_all_popups
```

Comment:
Yep, it's really tricky, but iSeries seems to be an special case and no PReP partition is needed. Take a look, for example, to this condition: https://github.com/imobach/yast-storage/blob/7fbee7ff1355ffd20d409c018a1b525bdb22469a/src/include/partitioning/custom_part_check_generated.rb#L374

On the other hand, installation is a parameter to this function.